internal/integrationtest/lib/lib_test.go

```diff
@@ -1587,3 +1587,40 @@ func TestLibBundlesWhenLibWithTheSameNameIsInstalledGlobally(t *testing.T) {
 		requirejson.Parse(t, stdout).Query(`.[].library.examples[1]`).MustContain(`"OTALeds"`)
 	}
 }
+
+func TestLibListDoesNotIncludeEmptyLibraries(t *testing.T) {
```

Comment:
In addition to the integration test could you make a unit test to check the `Load` function behavior here: https://github.com/arduino/arduino-cli/blob/master/arduino/libraries/libraries_test.go
After adding the unit-test you should be able to immediately see code coverage in the IDE.